app/controllers/order_items_controller.rb

```diff
@@ -14,4 +14,9 @@ def create
       redirect_to root_path, error: 'Something went wrong'
     end
   end
+
+  def show
+    @order_items = current_user.orders.last.shopping_cart_items
+    @totalprice = 0
```

Comment:
Please use proper snake_case for variables.  But also, can’t see that you actually use this one.

Reply:
Changed to total_price. It is used to calculate the total of the order in the order view.

Reply:
Changed to acts_as_shopping_cart built-in `subtotal` method (instead of `total`) as it avoids adding a default 8.25 pct tax